tests/strategies/report.py

```diff
@@ -135,6 +135,7 @@ def issues(draw):
         del extra["testCases"]
 
     return {
+        "uuid": None,
```

Comment:
Could we add a strategy here that picks from either `None` or `uuid`?
It should look something like
```python3
{"uuid": draw(sampled_from([none(), str(uuids(version=4))]}
```
Not sure whether this optional `None` could be formulated in a more efficient way. Building hypothesis tests is a bit trial and error sometimes, and I'm by no means an expert.

Reply:
Added None and uuid sampling